Mathlib/MeasureTheory/Function/StronglyMeasurable/AEStronglyMeasurable.lean

```diff
@@ -368,7 +368,7 @@ section Monoid
 variable {M : Type*} [Monoid M] [TopologicalSpace M] [ContinuousMul M]
 
 @[to_additive (attr := fun_prop, measurability)]
-theorem _root_.List.aestronglyMeasurable_prod' (l : List (α → M))
+theorem _root_.List.aestronglyMeasurable_fun_prod (l : List (α → M))
```

Comment:
I don't think that's the correct naming scheme: `fun_prod` should be used when there is `fun` somewhere, but there is no `fun` here. I'd call this one `aestronglyMeasurable_prod`, and the next one `aestronglyMeasurable_fun_prod`. Same thing below.

Reply:
https://leanprover-community.github.io/contribute/naming.html#unexpanded-and-expanded-forms-of-functions